packages/@sanity/types/src/schema/definition/type/common.ts

```diff
@@ -49,6 +65,7 @@ export interface SanityCreateOptions {
  *  */
 export interface BaseSchemaTypeOptions {
   sanityCreate?: SanityCreateOptions
+  canvasApp?: CanvasAppOptions
```

Comment:
Could you add at least one type test where `sanityCreate` is currently? Just nice to verify that its allowed by `defineType` et al